lib/zeromq/majordomo/asynchronous_arbiter.rb

```diff
@@ -0,0 +1,49 @@
+require 'ffi-rzmq'
+
+module Zeromq
+  module Majordomo
+    class AsynchronousArbiter
+      INFINITE = -1
+
+      class << self
```

Comment:
We could, I was trying to match the pattern of the other arbiters. Is there a reason that the rest of them are singletons?